src/components/DpButton/DpButton.vue

```diff
@@ -69,13 +69,14 @@ export default {
     },
 
     /**
-     * Variants of the button include `outline` (`text` to be implemented).
+     * Variants of the button. Possible values are `solid`, `outline`, and `subtle`.
      * When not specified, the default style (white on colored background) is applied.
      */
     variant: {
       required: false,
       type: String,
-      default: ''
+      default: 'solid',
+      validator: (prop) => ['solid', 'outline', 'subtle'].includes(prop)
```

Comment:
While we're at it, should we also add a validator for the `color` prop?

Reply:
right! b8c70ff121e704f8a479cb9ace370e7eb1aa4615